.CI/Test/Common.c

```diff
@@ -1,4 +1,4 @@
-#include "ModelicaUtilities.h"
+#include "../../Modelica/Resources/C-Sources/ModelicaUtilities.h"
```

Comment:
(Attaching comment to one of several possible places.)

Considering that _ModelicaUtilities.h_ has prototypes for functions that should be provided by the tool, and not for functions provided by the MSL, wouldn't a mode natural location be inside ModelicaServices?
```suggestion
#include "../../ModelicaServices/Resources/Include/ModelicaUtilities.h"
```

(Also note the use of the default include directory for the library.)

Reply:
As indicated in another place I think that proposed change would add some unnecessary problems; basically tools have to replace ModelicaServices with their own version - but we don't want to change the ModelicaUtilities.h used for building the libraries, or used for building other external libraries.
(Originally I also thought it might be a good idea, but I changed.)

Reply:
As per discussion during the [MAP-Lib meeting of 25/02/25](https://github.com/modelica/MA-Internal/blob/master/MA-Projects/MAP-LIB/MeetingMinutes/20250225-Monthly/README.md#pending-issues), we keep it Modelica/Sources/C-Sources. This make it clear that it's just a place where John Doe can look for the source code of the ModelicaUtilities headers when he needs it.